### DISCUSSION TITLE
Django dictionary extension

### DISCUSSION THREAD

**Discussion by Nosferatu31** [streetsidesoftware/cspell-dicts#2278] [Q&A]
There is a widely used Django library (`djangorestframework` aka DRF) that is not included in the Django dictionary.
Is it ok to add those terms to it or should the Django dictionary not include any terms from extensions/plugins?

**Comment by Jason3S** [comment]
@Nosferatu31,

The Django dictionary was generated from their website using a script.

I think it makes sense to add another word list for other popular Django related terms. Please make a PR with a new text file here: https://github.com/streetsidesoftware/cspell-dicts/tree/main/dictionaries/django/src

You can even add it from GitHub:
<img width="220" alt="image" src="https://github.com/streetsidesoftware/cspell-dicts/assets/3740137/155086f4-6122-4ebf-b14f-c698f371789d">
